### PR TITLE
Loading your Data (doc): revised text, figures.

### PR DESCRIPTION
**Pull request by BlazZupan**
I have changed some wording, revised figures due to now slightly different treatment of data files (Orange better figures out what is string and meta). Also, I now name continues and discrete as type, and meta and class as attribute kind.

### CODE REVIEW

File: doc/visual-programming/source/loading-your-data/index.rst [hunk 1]
```diff
@@ -2,77 +2,66 @@ Loading your Data
...
-denoting a gene class. The second column stores gene names, while the
-remaining columns record measurements that characterize each gene. Gene
-class can be used for classification. Gene name is a meta information, a
-label that is not relevant to any data mining algorithm, but can identify
-a data instance in, say, visualizations like scatter plot. We need to
-tell Orange that these first two columns are special. One way to do this
-within Orange is through `Select Columns <http://docs.orange.biolab.si/widgets/rst/data/selectattributes.html#select-attributes>`_ widget:
+Orange correctly assumed that a column with gene names is a meta information, which is **Data Table** displayed in columns shaded with light-gray. It has wrongly guessed theta `heat 20`, the last non-meta column in our data file, is a class column. To correct this in Orange, we can feed the data to the `Select Columns <http://docs.orange.biolab.si/widgets/rst/data/selectattributes.html#select-attributes>`_ widget:
```
Comment (kernc) [inline doc/visual-programming/source/loading-your-data/index.rst]: theta?

File: doc/visual-programming/source/loading-your-data/index.rst [hunk 2]
```diff
@@ -133,45 +118,41 @@ This is how the header with augmented attribute names looks like in
...
+Orange can read data from Google Sheets, as long as it conforms to the data presentation rules we have presented above. In Google Sheets, copy the shareable link (Share button, then Get shareable link) and paste it in the `Data File / URL` box of the File widget. For a taste, here's one such link you can use: `http://bit.ly/1J12Tdp <http://bit.ly/1J12Tdp>`_, and the way we have entered it in the **File** widget:
+
+.. image:: file-google-sheets.png
+    :width: 402 px
+    :align: center
+
+Data from LibreOffice
+---------------------
 
 If you are using LibreOffice, simply save your files in .xlsl format (available from the drop-down menu under *Save As Type*).
```
Comment (kernc) [inline doc/visual-programming/source/loading-your-data/index.rst]: It's .xls**x**, and we also support legacy .xls now.

File: doc/visual-programming/source/loading-your-data/index.rst [hunk 3]
```diff
@@ -2,77 +2,66 @@ Loading your Data
...
-    the optional element (**class**, **meta** or **string**).
+    mS\#name, i\#dummy.
+-   An alternative to the hash notation is Orange's native format with three
+    header rows: the first with attribute names, the second specifying
+    the type (**continuous**, **discrete** or **string**), and the third
+    proving information on the kind of attribute (**class**, **meta** or **string**).
 
-Data from Excel
----------------
+En Example: Data from Excel
```
Comment (lanzagar) [inline doc/visual-programming/source/loading-your-data/index.rst]: An Example

File: doc/visual-programming/source/loading-your-data/index.rst [hunk 4]
```diff
@@ -2,77 +2,66 @@ Loading your Data
...
-In Orange, let us start with a simple workflow with File and Data Table widgets,
+The file contains a header row, eight data instances (rows) and seven data attributes (columns). Empty cells in the table denote missing data entries. Rows represent genes; their function (class) is provided in the first column and their name in the second. The remaining columns store measurements that characterize each gene. With this data, we could, say, develop a classifier that would predict gene function from its characteristic measurements.
+
+Let us start with a simple workflow that reads the data and displays it in a table:
 
 .. image:: file-data-table-workflow.png
     :align: center
 
-and then load the data from Excel by opening the File widget (double click on the icon and of the widget) and click on the file browser icon ("..."),
+To load the data, open the File widget (double click on the icon and of the widget), click on the file browser icon ("...") and locate the downloaded file (from :download:`sample.xlsx <sample.xlsx>`) on your disk:
```
Comment (lanzagar) [inline doc/visual-programming/source/loading-your-data/index.rst]: on the icon and of the widget -> on the icon of the widget

File: doc/visual-programming/source/loading-your-data/index.rst [hunk 5]
```diff
@@ -2,77 +2,66 @@ Loading your Data
...
+To load the data, open the File widget (double click on the icon of the widget), click on the file browser icon ("...") and locate the downloaded file (from :download:`sample.xlsx <sample.xlsx>`) on your disk:
 
 .. image:: loadingyourdata.png
-    :width: 600 px
+    :width: 401 px
     :align: center
 
-locate the data file (e.g. :download:`sample.xlsx <sample.xlsx>`) and open
-it. The **File** widget sends data to **Data Table** widget, which displays the following result:
+The **File** widget sends the data to **Data Table** widget, which displays it in the spreadsheet. Double click the **Data File** widget to see its contents:
```
Comment (lanzagar) [inline doc/visual-programming/source/loading-your-data/index.rst]: - to _the_ **Data Table** widget
- if it was not mentioned before it should be "a spreadsheet"
- I also don't know if "spreadsheet" is the best word for the "table" in the widget?
(maybe this second part of the first sentence can be omitted)
- **Data File** widget -> **Data Table** widget?

File: doc/visual-programming/source/loading-your-data/index.rst [hunk 6]
```diff
@@ -133,45 +118,41 @@ This is how the header with augmented attribute names looks like in
 Excel (:download:`sample-head.xlsx <sample-head.xlsx>`):
 
 .. image:: spreadsheet-simple-head1.png
-    :width: 500 px
+    :width: 414 px
     :align: center
 
-We can again use a `Data Table <http://docs.orange.biolab.si/widgets/rst/data/datatable.html#data-table>`_ widget to read the data from Excel file. Orange will automatically recognize attribute values, which is evident in the modified class icons:
+We can again use again a **File** widget to load this data set and then render it in the **Data Table**:
```
Comment (lanzagar) [inline doc/visual-programming/source/loading-your-data/index.rst]: We can again use again a **File** widget -> We can again use a **File** widget

File: doc/visual-programming/source/loading-your-data/index.rst (hunk 5 above)
Comment (kernc) [inline doc/visual-programming/source/loading-your-data/index.rst]: "If antecedent, (← comma) consequent."

File: doc/visual-programming/source/loading-your-data/index.rst (hunk 6 above)
Comment (kernc) [inline doc/visual-programming/source/loading-your-data/index.rst]: data set → dataset

Comment (lanzagar) [inline doc/visual-programming/source/loading-your-data/index.rst]: Actually data set can be written as 2 words or 1:
https://en.wikipedia.org/wiki/Data_set

2 words is probably slightly more "proper" for now. It is common in English for phrases to start as 2 words and then through frequent usage they merge into a single word (often there is an intermediate version with a hyphen).

Anyway the most important thing is to be consistent and always use the same choice.

Comment (kernc) [inline doc/visual-programming/source/loading-your-data/index.rst]: Dataset [skipped](https://www.google.com/trends/explore#q=%22data%20set%22%2C%20%22dataset%22%2C%20%22data-set%22&cmpt=q&tz=Etc%2FGMT-1) the hyphenated stage. :stuck_out_tongue:

**Comment by kernc** [comment]
@BlazZupan When you're done changing, would you like to [squash these mini-fixups](https://www.google.si/search?q=github+squash+commits) into a single commit?

**Comment by BlazZupan** [comment]
Tega ne znam. Mi prosim napišeš kako. Hvala

File: doc/visual-programming/source/loading-your-data/index.rst [hunk 7]
```diff
@@ -2,77 +2,66 @@ Loading your Data
 =================
 
 Orange comes with its `own data format <http://docs.orange.biolab.si/reference/rst/Orange.data.formats.html#tab-delimited>`_, but can
-also handle Excel (.xlsl), comma or tab delimited data files. The input data
+also handle naitive Excel (.xlsx or .xls), comma- or tab-delimited data files. The input data
```
Comment (ajdapretnar) [inline doc/visual-programming/source/loading-your-data/index.rst]: native*

File: doc/visual-programming/source/loading-your-data/index.rst [hunk 8]
```diff
@@ -80,50 +69,46 @@ to a **Class** box, and attribute named `gene` to a **Meta Attribute**
...
+we will augment the names of the attributes with prefixes
+that define attribute type (continuous, discrete, string) and kind (class or meta attribute)
+Prefixes are separated from the attribute name with a hash sign ("\#"). Prefixes for the attribute kind are:
 
 -   c: class attribute
 -   m: meta attribute
 -   i: ignore the attribute
 
-and for the domain:
+and for type are:
```
Comment (ajdapretnar) [inline doc/visual-programming/source/loading-your-data/index.rst]: Omit 'are'.

**Pull request merged**